xchange-blockchain/src/main/java/org/knowm/xchange/blockchain/dto/account/BlockchainSymbols.java

```diff
@@ -0,0 +1,32 @@
+package org.knowm.xchange.blockchain.dto.account;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+import org.knowm.xchange.currency.Currency;
+
+@Data
+@Builder
+@Jacksonized
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BlockchainSymbols {
+    private final Currency base_currency;
```

Comment:
Please use cammelCase. You can use @JsonProperty to put the json name (see for example BlockchainTicker